src/yamlToJSON.cc

```diff
@@ -0,0 +1,129 @@
+/* ========================================================================== *
+ *
+ * @file yamlToJSON.cc
+ *
+ * @brief Convert a YAML string to a JSON object.
+ *
+ *
+ * -------------------------------------------------------------------------- */
+
+#include <string>
+
+#include <nlohmann/json.hpp>
+#include <yaml-cpp/yaml.h>
+
+#include "flox/core/exceptions.hh"
+
+
+/* -------------------------------------------------------------------------- */
+
+namespace flox {
+
+/* -------------------------------------------------------------------------- */
+
+  nlohmann::json
+yamlToJSON( std::string_view yaml )
+{
+  std::function<void( nlohmann::json &, const YAML::Node & )> visit;
+
+  visit = [&]( nlohmann::json & jto, const YAML::Node & yfrom )
+    {
+      switch( yfrom.Type() )
+        {
+          case YAML::NodeType::Null:
+            jto = nullptr;
```

Comment:
Same question here, is there a better type for this?

Reply:
see answer above.